src/shared/utils/state.ts

```diff
@@ -45,7 +47,7 @@ export const getFulfilledData = <T extends Record<string, BaseSchema>, K>({
     ) {
       selectedProperty.requestId = initialState[key].requestId;
       selectedProperty.status = 'success';
-      selectedProperty.data = action.payload.data;
+      selectedProperty.data = mapper(action.payload.data);
```

Comment:
it's redundant loop when `mapper` is not provided. Maybe it's better just return `action.payload.data` ?